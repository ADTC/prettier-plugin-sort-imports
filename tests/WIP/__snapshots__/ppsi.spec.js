run_spec(__dirname, ["typescript"], {
    importOrder: ["^react",
    "<THIRD_PARTY_MODULES>",
    "",
    "^@(.*)",
    "^[./]"],
    importOrderSeparation: false,
    "trailingComma": "none",
});
