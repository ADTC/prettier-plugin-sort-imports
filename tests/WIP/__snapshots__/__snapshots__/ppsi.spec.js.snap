// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test.ts - typescript-verify: test.ts 1`] = `
import React from 'react';
import clsx from 'clsx';
import Link from '@docusaurus/Link';
import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
import HomepageFeatures from '@site/src/components/HomepageFeatures';
import Layout from '@theme/Layout';
import styles from './index.module.css';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import React from "react";
import clsx from "clsx";

import Link from "@docusaurus/Link";
import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
import HomepageFeatures from "@site/src/components/HomepageFeatures";
import Layout from "@theme/Layout";
import styles from "./index.module.css";

`;
